### DISCUSSION TITLE
Pinned application

### DISCUSSION THREAD

**Discussion by dikiaap** [davatorium/rofi#1761] [Q&A] [answered]
Is there a way to "pin" specific applications to the top of the list when we run "rofi -show drun" so they won't be reordered after we choose them?

**Comment by DaveDavenport** [comment]
There is no way to pin applications.   The 'favorites' system can be disabled by disabling history (see manpage).

**Comment by dikiaap** [comment, marked as answer]
I actually want them as shortcuts so I can easily open them. For now, my workaround is just increasing the number in `~/.cache/rofi3.druncache`, although the number will keep changing.

**Reply by DaveDavenport** [reply]
A work around is to make a small 'script' mode that list your favorites, then combine this with drun using the combi modi.
